cardano-cli/src/Cardano/CLI/Run/Friendly.hs

```diff
@@ -162,5 +166,20 @@ friendlyTxOutValue = \case
   TxOutAdaOnly _ lovelace -> friendlyLovelace lovelace
   TxOutValue _ multiasset -> toJSON multiasset
 
+friendlyMetadata :: TxMetadataInEra era -> Aeson.Value
```

Comment:
Let's undo the alignment as per the style document.

Reply:
just for `friendlyMetadata` function or for the whole file?

Reply:
done

Reply:
There is still aligned code @cblp

Reply:
done now?